hpo-case-annotator-gui/src/main/java/org/monarchinitiative/hpo_case_annotator/gui/controllers/variant/VariantUtil.java

```diff
@@ -192,43 +190,69 @@ public static void geneAccessionNumberGrabber(String entrezGeneId,GenomeAssembly
 
             String resp = response.toString().replaceAll("\\s+", "");
             // This is XML but all we need are the NM_12345 accession numbers as follows
-            Pattern pat = Pattern.compile("<Gene-commentary_accession>(NM_\\d+)</Gene-commentary_accession>");
-            Matcher mat = pat.matcher(resp);
-            while (mat.find()) {
-                String refseqAccession = mat.group(1);
-                builder.add(refseqAccession);
+            Pattern accessionPattern = Pattern.compile("<Gene-commentary_accession>(NM_\\d+)</Gene-commentary_accession>");
+            Matcher accessionMatcher = accessionPattern.matcher(resp);
+            String refseqAccession;
+            if (accessionMatcher.find()) {
+                refseqAccession = accessionMatcher.group(1);
+            } else {
+                LOGGER.warn("Unable to find accession id for gene 'HGNC:{}'", entrezGeneId);
+                return;
+            }
+
+            // Then we extract the accession number version
+            // TODO(pnrobinson) - please figure out how to make this work.
```

Comment:
@pnrobinson Could you please fix this code so that it would be able to identify version of the transcript accession id?
If the transcript is `NM_004004.4`, we currently have the `NM_004004` part. However, I was not able to parse the XML content to get the version (`4` in this case0